pengine/test10/rebalance-unique-clones.xml

```diff
@@ -0,0 +1,175 @@
+        <nvpair id="cib-bootstrap-options-last-lrm-refresh" name="last-lrm-refresh" value="1507206008"/>
+        <nvpair id="cib-bootstrap-options-placement-strategy" name="placement-strategy" value="default"/>
+      </cluster_property_set>
+    </crm_config>
+    <nodes>
+      <node id="169745988" uname="web1">
+        <instance_attributes id="nodes-169745988">
+          <nvpair id="nodes-169745988-webcluster" name="webcluster" value="1"/>
+        </instance_attributes>
+      </node>
+      <node id="169745990" uname="web2">
+        <instance_attributes id="nodes-169745990">
+          <nvpair id="nodes-169745990-webcluster" name="webcluster" value="1"/>
+          <nvpair id="nodes-169745990-standby" name="standby" value="off"/>
+        </instance_attributes>
+      </node>
+    </nodes>
+    <resources>
+      <clone id="cl.vip.web">
+        <meta_attributes id="cl.vip.web-meta_attributes">
+          <nvpair name="globally-unique" value="true" id="cl.vip.web-meta_attributes-globally-unique"/>
+          <nvpair name="clone-max" value="2" id="cl.vip.web-meta_attributes-clone-max"/>
+          <nvpair name="clone-node-max" value="2" id="cl.vip.web-meta_attributes-clone-node-max"/>
+          <nvpair name="ordered" value="true" id="cl.vip.web-meta_attributes-ordered"/>
+          <nvpair name="interleave" value="false" id="cl.vip.web-meta_attributes-interleave"/>
+          <nvpair id="cl.vip.web-meta_attributes-target-role" name="target-role" value="Started"/>
+        </meta_attributes>
+        <primitive id="vip.web" class="ocf" provider="heartbeat" type="IPaddr2">
+          <instance_attributes id="vip.web-instance_attributes">
+            <nvpair name="ip" value="192.168.195.110" id="vip.web-instance_attributes-ip"/>
+            <nvpair name="cidr_netmask" value="32" id="vip.web-instance_attributes-cidr_netmask"/>
+            <nvpair name="iflabel" value="web" id="vip.web-instance_attributes-iflabel"/>
+            <nvpair name="clusterip_hash" value="sourceip-sourceport" id="vip.web-instance_attributes-clusterip_hash"/>
+            <nvpair name="mac" value="03:00:60:0D:F0:0D" id="vip.web-instance_attributes-mac"/>
+          </instance_attributes>
+          <operations>
+            <op name="monitor" interval="10s" timeout="20s" id="vip.web-monitor-10s"/>
+            <op name="start" interval="0s" timeout="20s" id="vip.web-start-0s"/>
+            <op name="stop" interval="0s" timeout="20s" id="vip.web-stop-0s"/>
+          </operations>
+          <meta_attributes id="vip.web-meta_attributes">
+            <nvpair name="resource-stickiness" value="0" id="vip.web-meta_attributes-resource-stickiness"/>
+          </meta_attributes>
+        </primitive>
+      </clone>
+      <clone id="cl.httpd.web">
+        <meta_attributes id="cl.httpd.web-meta_attributes">
+          <nvpair name="globally-unique" value="true" id="cl.httpd.web-meta_attributes-globally-unique"/>
```

Comment:
This one is probably not globally-unique.

Reply:
Correct, that was an oversight in the original test case. I probably should have changed it to false for the regression test, but since clone-node-max=1, it doesn't affect the outcome, only the status output format.

Reply:
BTW thanks for the close review. We could always use more eyes on pull requests.